lib/nexthop.c

```diff
@@ -759,12 +759,33 @@ uint32_t nexthop_hash_quick(const struct nexthop *nexthop)
 			key = jhash_1word(nexthop->backup_idx[i], key);
 	}
 
-	if (nexthop->nh_srv6) {
-		key = jhash_1word(nexthop->nh_srv6->seg6local_action, key);
-		key = jhash(&nexthop->nh_srv6->seg6local_ctx,
-			    sizeof(nexthop->nh_srv6->seg6local_ctx), key);
-		key = jhash(&nexthop->nh_srv6->seg6_segs,
-			    sizeof(nexthop->nh_srv6->seg6_segs), key);
+	if (nexthop && nexthop->nh_srv6) {
```

Comment:
How is nexthop not already derefed in every path up to this point?  Please remove

Reply:
done.